src/consensus/pbft_manager.cpp

```diff
@@ -1284,16 +1165,18 @@ std::vector<std::vector<uint>> PbftManager::createMockTrxSchedule(
   return blocks_trx_modes;
 }
 
-std::pair<blk_hash_t, bool> PbftManager::identifyLeaderBlock_(std::vector<Vote> const &votes) {
+std::pair<blk_hash_t, bool> PbftManager::identifyLeaderBlock_(std::vector<std::shared_ptr<Vote>> votes) {
```

Comment:
even worse than before, instead of const ref to vector now it is a copy of vector

Reply:
is it better to call `vote_mgr_->getProposalVotes(round)` inside this function as it is used only once like this:
```    
auto proposal_votes = vote_mgr_->getProposalVotes(round);
std::pair<blk_hash_t, bool> leader_block = identifyLeaderBlock_(proposal_votes);
```

Reply:
Moved vote_mgr_->getProposalVotes(round) inside identifyLeaderBlock_, thanks